setup.py

```diff
@@ -27,33 +27,35 @@
     ext_files.append('simhash/simhash.cpp')
 
 ext_modules = [
-    Extension('simhash.simhash', ext_files,
+    Extension(
+        'simhash.simhash', ext_files,
         language='c++',
         extra_compile_args=['-std=c++11'],
         include_dirs=['simhash/simhash-cpp/include']
     )
 ]
 
-setup(name           = 'simhash',
-    version          = '0.2.0',
-    description      = 'Near-Duplicate Detection with Simhash',
-    url              = 'http://github.com/seomoz/simhash-py',
-    author           = 'Dan Lecocq',
-    author_email     = 'dan@moz.com',
-    classifiers      = [
+setup(
+    name='simhash',
+    version='0.3.0',
+    description='Near-Duplicate Detection with Simhash',
+    url='http://github.com/seomoz/simhash-py',
+    author='Dan Lecocq',
+    author_email='dan@moz.com',
```

Comment:
It's unclear who should be the author at this point. I'm also not sure it really matters. I didn't feel the need to change it at the moment.

Honestly, I'm not sure what the best thing should be for public repos, we probably don't want to have an individual email, but we also don't want to publish internal mailing list names, so it's kind of a toss up.